data/videos.yml

```diff
@@ -61,3 +61,52 @@ KlMlZTHShFw:
   description: I highly recommend the entire talk, as well as Michael Albert's book and other work on Participatory Economics
   series: "Michael Albert"
   source: youtube
+vk5xnEL8mYg:
+  id: vk5xnEL8mYg
+  title: "All Cops Are Bad"
+  channel: thoughtslime
+  description: Ok so this one is gonna make some people pretty cheesed. Look, the police are bad, and it's time we talked about why, and what we should do about it.
+  series: thoughtslime
+  source: youtube
+Yw_RQxNM0Kk:
+  id: Yw_RQxNM0Kk
+  title: "DO #BLUELIVESMATTER ?"
+  channel: katblaque
+  description: In this video, I talk about Blue Lives Matter, Police Brutality and violence against the police. I also share my personal perspective on Law Enforcement.
+  series: katblaque
+  source: youtube
+LB3HUXdmid4:
+  id: LB3HUXdmid4
+  title: "Let's talk about police militarization...."
+  channel: beauofthefifthcolumn
+  description:
```

Comment:
Probably want to throw in a description here for consistencies sake.

Reply:
@craiglonsdale agreed we should be striving for that, but it's hard when the source video doesn't have a description, we don't have a process for handling this yet. I think we can merge and add later. There is already an EarthStrike video without a description.

Reply:
Ok fair enough. Maybe we can do something with the styling to handle no description in the future.